### DISCUSSION TITLE
将域名后缀单独写一行会不会有风险？

### DISCUSSION THREAD

**Discussion by Benjamin1919** [v2fly/domain-list-community#1958] [Q&A] [answered]
引用一段v2ray官方配置 (https://www.v2ray.com/chapter_02/03_routing.html#ruleobject) 的原话：
`纯字符串: 当此字符串匹配目标域名中任意部分，该规则生效。比如"sina.com"可以匹配"sina.com"、"sina.com.cn"和"www.sina.com"，但不匹配"sina.cn"。`

而在官方的geosite:google标签 (https://github.com/v2fly/domain-list-community/blob/master/data/google) 的域名列表里，对于.and后缀的域名只写了and：
```
# All .and domains
and
```
那么理论上"and"既可以匹配"*.and"这种子域名形式，也可以匹配"and.com"、"and.cn"之类的域名，这样岂不是把并不属于谷歌的域名也给囊括进来了？

是不是我理解有误？麻烦解答一下，谢谢！

**Comment by him007** [comment, marked as answer]
Subdomain begins with domain:, followed by a valid domain name. The prefix domain: may be omitted.
子域以domain：开头，后跟有效域名。domain：可以省略。
# All .and domains
and
等同于domain:and，表示匹配后缀为and的域名
